docs/standard/framework-libraries.md

```diff
@@ -1,37 +1,41 @@
 ---
 title: Framework Libraries
-description: Learn how libraries provide implementations for many general and app-specific types, algorithms, and utility functionality.
+description: Learn how .NET libraries provide implementations for many general and app-specific types, algorithms, and utility functionality.
 author: richlander
-ms.date: 06/20/2016
+ms.date: 10/06/2021
 ms.assetid: 7b77b6c1-8367-4602-bff3-91e4c05ac643
 ---
-# Framework Libraries
+# Framework libraries
 
-.NET has an expansive standard set of class libraries, referred to as either the base class libraries (core set) or framework class libraries (complete set). These libraries provide implementations for many general and app-specific types, algorithms and utility functionality. Both commercial and community libraries build on top of the framework class libraries, providing easy to use off-the-shelf libraries for a wide set of computing tasks.
+.NET has an expansive standard set of class libraries, referred to as either the base class libraries (core set) or framework class libraries (complete set). These libraries provide implementations for many general and app-specific types, algorithms, and utility functionality. Both commercial and community libraries build on top of the framework class libraries, providing easy to use, off-the-shelf libraries for a wide set of computing tasks.
 
-A subset of these libraries are provided with each .NET implementation. Base Class Library (BCL) APIs are expected with any .NET implementation, both because developers will want them and because popular libraries will need them to run. App-specific libraries above the BCL, such as ASP.NET, will not be available on all .NET implementations.
+A subset of these libraries are provided with each .NET implementation. Base class library (BCL) APIs are expected with any .NET implementation, both because developers will want them and because popular libraries will need them to run. App-specific libraries above the BCL, such as ASP.NET, will not be available on all .NET implementations.
 
-## Base Class Libraries
+[NuGet](https://www.nuget.org/) is a package manager built specifically for .NET that contains over 90,000 packages.
```

Comment:
This seems out of place here, doesn't obviously fit in with either the preceding or the following text.